parser/testdata/workflows/greetings-v08-spec.sw.yaml

```diff
@@ -14,17 +14,91 @@
 
 id: custom.greeting
 version: '1.0'
-description: Greet Someone
 specVersion: "0.8"
+description: Greet Someone
+# name: Greeting example #can be empty
+# start: Greet #can be empty
 functions:
   - name: greetingCustomFunction
     operation: /path/to/my/script/greeting.ts#CustomGreeting
+    # Support custom function type definition
     type: custom
   - name: sendTextFunction
     operation: http://myapis.org/inboxapi.json#sendText
-
+    type: graphql
+  - name: greetingFunction
+    operation: file://myapis/greetingapis.json#greeting
 states:
-  - name: Greet
+  - name: GreetDelay
+    type: delay
+    timeDelay: PT5S
+    transition:
+      nextState: StoreCarAuctionBid
+  - name: StoreCarAuctionBid
+    type: event
+    exclusive: true
+    onEvents:
+      - eventRefs:
+          - CarBidEvent
+        eventDataFilter:
+          useData: true
+          data: "test"
+          toStateData: "testing"
+        actionMode: parallel
+        actions:
+          - functionRef:
+              refName: StoreBidFunction
+              arguments:
+                bid: "${ .bid }"
+    timeouts:
+      eventTimeout: PT1H
+      actionExecTimeout: PT3S
+      stateExecTimeout:
+        total: PT1S
+        single: PT2S
+  - name: ParallelExec
+    type: parallel
+    completionType: allOf
+    branches:
+      - name: ShortDelayBranch
+        actions:
+          - subFlowRef: shortdelayworkflowid
+        timeouts:
+          actionExecTimeout: "PT5H"
+          branchExecTimeout: "PT6M"
+      - name: LongDelayBranch
+        actions:
+          - subFlowRef: longdelayworkflowid
+    timeouts:
+      branchExecTimeout: "PT6M"
+      stateExecTimeout:
+        total: PT1S
+        single: PT2S
+  - name: CheckVisaStatusSwitchEventBased
+    type: switch
+    eventConditions:
+      - eventRef: visaApprovedEvent
+        transition: HandleApprovedVisa
+      - eventRef: visaRejectedEvent
+        transition: HandleRejectedVisa
+    eventTimeout: PT1H
+    stateExecTimeout:
+      total: PT1S
+      single: PT2S
```

Comment:
eventTimeout & stateExecTimeout is under `timeouts` property like other state.

Reply:
yep, copied from others json examples from the tests ones, seems that they were wrong as well.
thanks for the heads up.